provider/redshift.go

```diff
@@ -67,7 +67,9 @@ func (q redshiftSQLQueries) registerResources(db *sql.DB, tableName string, sche
 			sanitize(schema.Entity), sanitize(schema.Value), time.UnixMilli(0).UTC(), sanitize(schema.SourceTable))
 	}
 	if _, err := db.Exec(query); err != nil {
-		return err
+		wrapped := fferr.NewExecutionError(pt.RedshiftOffline.String(), err)
+		wrapped.AddDetail("table_name", tableName)
```

Comment:
Would we want to add the query as a detail or think thats too much?

Reply:
I've been avoiding adding queries to details given the possible size of them.